pkgs/applications/science/electronics/gerbv/default.nix

```
@@ -21,15 +20,19 @@ stdenv.mkDerivation {
   ];
 
   nativeBuildInputs = [ autoreconfHook pkg-config automake autoconf ];
-  buildInputs = [ gettext libtool cairo gtk2 ];
+  buildInputs = [ gettext libtool cairo gtk2-x11.dev ];
 
   configureFlags = ["--disable-update-desktop-database"];
 
+  CFLAGS = [
+    "-Wno-format-security"
+  ];
```

Does the following work for you?

```suggestion
  env.NIX_CFLAGS_COMPILE = "-Wno-format-security";
```

---

Yep, works for me. Out of curiosity, I did some digging and I see [discussion on about phasing out NIX_CFLAGS_COMPILE](https://github.com/NixOS/nixpkgs/issues/79303). Is there a source on where I can learn about the nuance between them?

---

I consider `env.NIX_CFLAGS_COMPILE` to be more uniform. Other environmental variables could of course work here or there.